App is getting crash due to java.lang.RuntimeException on WelcomeActivity

**Steps to reproduce:** 

By simply launching the application.

**Add system logs:**

AndroidRuntime: FATAL EXCEPTION: main
                                                                                   Process: openfoodfacts.github.scrachx.openfood, PID: 990
                                                                                   java.lang.RuntimeException: Unable to start activity ComponentInfo{openfoodfacts.github.scrachx.openfood/openfoodfacts.github.scrachx.openfood.views.WelcomeActivity}: android.util.AndroidRuntimeException: requestFeature() must be called before adding content
                                                                                       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2455)
                                                                                       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515)
                                                                                       at android.app.ActivityThread.access$1000(ActivityThread.java:154)
                                                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379)
                                                                                       at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                       at android.os.Looper.loop(Looper.java:157)
                                                                                       at android.app.ActivityThread.main(ActivityThread.java:5555)
                                                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745)
                                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635)
                                                                                    Caused by: android.util.AndroidRuntimeException: requestFeature() must be called before adding content
                                                                                       at com.android.internal.policy.PhoneWindow.requestFeature(PhoneWindow.java:317)
                                                                                       at android.app.Activity.requestWindowFeature(Activity.java:3759)
                                                                                       at openfoodfacts.github.scrachx.openfood.views.WelcomeActivity.onCreate(WelcomeActivity.java:46)
                                                                                       at android.app.Activity.performCreate(Activity.java:6357)
                                                                                       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
                                                                                       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408)
**Expected behaviour:** 

welcome screen should have appeared.

**Observed behaviour:** 

The app is getting crash.

**Device and Android version:** 

Android 6

I am working on it.

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/WelcomeActivity.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/WelcomeActivity.java
@@ -32,7 +32,10 @@ public class WelcomeActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_welcome);
 
         prefManager = new PrefManager(this);
         if (!prefManager.isFirstTimeLaunch()) {
@@ -43,9 +46,8 @@ public class WelcomeActivity extends AppCompatActivity {
         if (Build.VERSION.SDK_INT >= 21) {
             getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_STABLE | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN);
         }
-        requestWindowFeature(Window.FEATURE_NO_TITLE);
-        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);
-        setContentView(R.layout.activity_welcome);
+        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
 
         viewPager = (ViewPager) findViewById(R.id.view_pager);
         dotsLayout = (LinearLayout) findViewById(R.id.layoutDots);

